draft-ietf-acme-acme.md

```diff
@@ -2002,7 +1999,13 @@ propagating across a cluster or firewall rules not being in place.
 
 Clients SHOULD NOT respond to challenges until they believe that the server's
 queries will succeed. If a server's initial validation query fails, the server
-SHOULD retry the query after some time.  While the server is still trying, the
+SHOULD retry the query after some time, in order to account for delay in setting 
+up responses such as DNS records or HTTP resources.  The precise retry schedule
+is up to the server, but server operators should keep in mind the operational
+scenarios that the schedule is trying to accommodate.  For example, a server
+might perform the first few of retries at intervals 5 or 10 seconds to catch
```

Comment:
typo: "might perform the first few of retries". Maybe "might perform the first few retries" instead?

nit: "at intervals 5 or 10 seconds" -> "after an interval of 5 or 10 seconds" reads more naturally to me.

Reply:
I also think the phrasing "to catch automated clients" is strange. Catch them doing what?

Reply:
Also there's a double sapce before "For example, a server"

Reply:
typo - fixed
"at intervals" - I think this is correct usage ("after an interval" would mean "wait 5-10s, then as quickly as you want")
"to catch" -> "to provide a prompt response for"